src/main/java/de/tum/in/www1/artemis/service/CourseScoreCalculationService.java

```diff
@@ -51,10 +51,12 @@ record MaxAndReachablePoints(double maxPoints, double reachablePoints) {
 
     /**
      * Calculates max and reachable max points for the given exercises.
-     * Max points are the sum of all included (see {@link #includeIntoScoreCalculation(Exercise)}) exercises, whose due date is over or unset or who are automatically assessed and
-     * assessment is done (see {@link #isAssessmentDone(Exercise)}).
-     * Reachable max points are the sum of all included and already assessed exercises.
-     * Example: An automatically assessed programming exercise that has the buildAndTestStudentSubmissionsAfterDueDate set in the future is included in the max points calculation,
+     * Max points are the sum of the points for all included (see {@link #includeIntoScoreCalculation(Exercise)}) exercises, whose due date is over or unset or who are
+     * automatically assessed and
+     * the buildAndTestStudentSubmissionsAfterDueDate is in the past.
```

Comment:
Sorry but something happened here.

```suggestion
     * automatically assessed and the buildAndTestStudentSubmissionsAfterDueDate is in the past.
```